metadata.json

```diff
@@ -13,6 +13,12 @@
   ],
   "issues_url": "https://github.com/voxpupuli/puppet-puppetboard/issues",
   "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
```

Comment:
can you please also add rocky/alma/oracle 9?

Reply:
Added them.